packages/gestalt/src/Video/Controls.js

```diff
@@ -131,18 +131,18 @@ function VideoControls({
 
   const muted = volume === 0;
 
-  const [showFullscreenButton, setShowFullscreenButton] = useState(false);
+  const [showFullscreenButton, setShowFullscreenButton] = useState<boolean>(false);
 
   useEffect(() => {
     setShowFullscreenButton(
       typeof document !== 'undefined' &&
-        (document.fullscreenEnabled ||
+        (!!document.fullscreenEnabled ||
```

Comment:
Since the `document` checks are already wrapped in parens, you should be able to coerce the entire statement once instead of each `document` check